default/test_package/conanfile.py

```diff
@@ -1,8 +1,6 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
```

Comment:
@SSE4 is a vim user, he will deny your PR because of this :joy: Put emacs suppport instead :smiling_imp:

Reply:
And on his birthday :smiling_imp: 

This got past: https://github.com/bincrafters/conan-templates/commit/104f5ba53c1ac22304fccff14b710805b40c2165

Reply:
this is okay, no objections :)